src/client/app/components/TimeZoneSelect.tsx

```diff
@@ -4,42 +4,35 @@
 
 import * as React from 'react';
 import Select from 'react-select';
-import axios from 'axios';
-import { TimeZones, TimeZoneOption } from 'types/timezone';
+import { TimeZoneOption } from 'types/timezone';
 import translate from '../utils/translate';
+import * as moment from 'moment-timezone';
 
 interface TimeZoneSelectProps {
 	// The timezone is a string and null is stored in DB when there isn't one.
 	current: string | null;
 	handleClick: (value: string) => void;
 }
 
-let options: null | TimeZoneOption[] = null;
-
 const TimeZoneSelect: React.FC<TimeZoneSelectProps> = ({ current, handleClick }) => {
 
-	const [optionsLoaded, setOptionsLoaded] = React.useState(false);
-
-	React.useEffect(() => {
-		if (!optionsLoaded) {
-			axios.get('/api/timezones').then(res => {
-				const timeZones = res.data;
-				const resetTimeZone = [{value: null, label: translate('timezone.no')}];
-				const allTimeZones = (timeZones.map((timezone: TimeZones) => {
-					return { value: timezone.name, label: `${timezone.name} (${timezone.abbrev}) ${timezone.offset}` };
-				}));
-				options = [...resetTimeZone, ...allTimeZones];
-				setOptionsLoaded(true);
-			});
-		}
-	}, []);
+	const getTimeZones = () => {
+		const zoneNames = moment.tz.names();
+		return zoneNames.map(zoneName => {
+			const zone = moment.tz(zoneName);
+			const abbrev = zone.format('z');
+			return { value: zoneName, label: `${zoneName} (${abbrev})` };
+		});
+	};
+	const resetTimeZone = [{ value: null, label: translate('timezone.no') }];
+	const options = [...resetTimeZone, ...getTimeZones()];
 
 	const handleChange = (selectedOption: TimeZoneOption) => {
 		handleClick(selectedOption.value);
 	};
 
 	return (options !== null ?
```

Comment:
The need to check for null options was a result of the way the data was requested from the database. With it now being gotten from moment this cannot happen. I removed the conditional and used:
	return <Select isClearable={false} value={options.filter(({ value }) => value === current)} options={options} onChange={handleChange} />;
It seems to work fine. I think this change should be moade if @Jake-Bodin does not find any issues with is.

Reply:
I don't see any changes made from my original code, I don't feel I know enough to know if the new timezones are properly effecting the meters but i don't see any changes or problems.

Reply:
Sorry this was not clear. The comment had the desired code. I made this change and push a commit to put it into the PR. This PR should now be ready to merge.